semantic/src/main/kotlin/tools/samt/semantic/Package.kt

```diff
@@ -0,0 +1,73 @@
+package tools.samt.semantic
+
+import tools.samt.parser.BundleIdentifierNode
+import tools.samt.parser.IdentifierNode
+import tools.samt.parser.NamedDeclarationNode
+
+class Package(val name: String) {
```

Comment:
Why is this a separate class and not just the `PackageType` class from the type model?

Reply:
Currently the `Package` is used in the first step of building the package directory structure and the `PackageType` is then used during the type resolution - we could merge them, but I haven't focussed on that for now